.github/workflows/build-executable.yaml

```
@@ -57,26 +60,30 @@
+  MakeBuild:
+    name: Make Build
     needs:
       - check_branch
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
         with:
           fetch-depth: 0
-      - name: Set up JDK 21
+      - name: Set up Java and Go
         uses: actions/setup-java@v2
```

actionlint

the runner of "actions/setup-java@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)